### DISCUSSION TITLE
Image management

### DISCUSSION THREAD

**Discussion by wilonweb** [thuliteio/doks#1169] [Q&A]
Hello

When i put an image on my post.md 
```
{{< picture src="./8biticon.png" alt="A bird flying through a field of tall grass" >}}
```
( i find a solution to keep the original size with html import and the static folder. )
But if i use picture my image is croped and i would like know how can i choose the width and height. 

Second question. 

On the post if i put the image on a folder images like 
```
{{< picture src="./images/8biticon.png" alt="A bird flying through a field of tall grass" >}}
```
I got an error
```
The "picture" partial was unable to get "./images/8biticon.png" in blog\shortcodes.md  "C:\Users\wilfr\Documents\Code\Hugo\doks\doks-test\content\blog\shortcodes.md:99:1": failed to render shortcode "picture": failed to process shortcode: "C:\Users\wilfr\Documents\Code\Hugo\doks\doks-test\node_modules\@hyas\images\layouts\shortcodes\picture.html:125:4": execute of template failed at <partial "picture.html" $ctx>: error calling partial: "C:\Users\wilfr\Documents\Code\Hugo\doks\doks-test\node_modules\@hyas\images\layouts\partials\picture.html:203:13": execute of template failed at <$bi.Height>: can’t evaluate field Height in type string 
``` 

Do you know why ?

**Comment by h-enk** [comment]
What if you omit the `./` from `src` — see also [Resource Locations](https://images.gethyas.com/docs/basics/resources/)

> But if i use picture my image is croped and i would like know how can i choose the width and height.

You can use the `width` parameter — see for example [Image Orientation](https://images.gethyas.com/docs/built-ins/image-orientation/)

More options (like for example aspect ratios) will be available in the upcoming release of Hyas Images — available soon

**Comment by encient** [comment]
> ( i find a solution to keep the original size with html import and the static folder. )

Hi bro, I'm having issue to keep my image as original size in my blog and doc page, can you share how you did it?

**Reply by wilonweb** [reply]
I put the image in the static folder, and i call it with html
